virtio-queue/src/queue/verification.rs

```diff
@@ -459,6 +459,17 @@ impl kani::Arbitrary for ProofContext {
     }
 }
 
+/// Helper function to determine if `point` is in the wrapping range `[start, stop)`.
+fn pred_in_wrapping_range(start: Wrapping<u16>, stop: Wrapping<u16>, point: Wrapping<u16>) -> bool {
+    // Check if point is in the range [start, stop) in a wrapping manner.
+    if start <= stop {
+        start <= point && point < stop
+    } else {
+        // If start > stop, we wrap around the maximum value of u16.
+        point >= start || point < stop
+    }
+}
+
 /// # Specification (VirtIO 1.3, Section 2.7.7.2: "Device Requirements: Used Buffer Notification Suppression")
```

Comment:
The commit https://github.com/rust-vmm/vm-virtio/pull/363/commits/4a8b048604ee45dfcdc3e5219161e35b249a1b89 description is not really clear to me.

It seems now this function don't test only the suppression, so should we update this documentation?

Reply:
I think the proof is testing both w/wo suppression. I think the comment should only mention about the notification.